vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/java/com/vaadin/flow/component/spreadsheet/SpreadsheetFilterTable.java

```diff
@@ -21,12 +21,10 @@
 
 import org.apache.poi.ss.usermodel.Sheet;
 import org.apache.poi.ss.util.CellRangeAddress;
-
 import com.vaadin.flow.component.ClickEvent;
 import com.vaadin.flow.component.Component;
 import com.vaadin.flow.component.ComponentEvent;
 import com.vaadin.flow.component.ComponentEventListener;
-import com.vaadin.flow.component.HasComponents;
 import com.vaadin.flow.component.button.Button;
 import com.vaadin.flow.component.orderedlayout.VerticalLayout;
```

Comment:
I'm wondering if we need to add vertical layout web component here, I guess we can just use the div container for adding the checkboxes, and then  adding some extra flex css to the div. Something like the following in `ItemFilter`  (the same should be used in the `SpreadsheetFilterTable`)

```
    protected void initLayouts() {
        add(allItems);
        add(filterCheckbox);
        setClassName(ITEM_FILTER_LAYOUT_CLASSNAME);
        // Better in CSS file
        getStyle().set("display", "flex");
        getStyle().set("flex-direction", "column");
    }
```
It would make the table display always the same, when the user custom their Lumo theme